shelley/chain-and-ledger/shelley-spec-ledger-test/test/Test/Shelley/Spec/Ledger/Examples/Consensus.hs

```diff
@@ -0,0 +1,539 @@
+{-# LANGUAGE ConstraintKinds #-}
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE TypeFamilies #-}
+{-# OPTIONS_GHC -Wno-orphans #-}
+
+module Test.Shelley.Spec.Ledger.Examples.Consensus where
```

Comment:
This is in the wrong place - if it's something which should be exported in the library, it should live in `src`, not in `test`

Reply:
Indeed. Moved it to `src` instead.